### PR TITLE
Strip down dependencies

### PR DESCRIPTION
**Pull request by Domi04151309**
Removes some unused code and dependencies

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -1,11 +1,11 @@
 apply plugin: 'com.android.application'
 
 android {
-    compileSdkVersion 29
+    compileSdkVersion 30
```
Comment (92lleo) [inline app/build.gradle:4]: Fdroid does not support SDK 30 yet. To avoid failing builds I'd wait till fdroid updates

Comment (Domi04151309) [reply app/build.gradle:4]: Oh i didn't know that

Comment (92lleo) [reply app/build.gradle:4]: You could remove the sdk bump to get it merged right away or I'll check in here again once sdk 30 is available.

**Comment by 92lleo** [comment]
Sorry for my slow response time. Can you edit the "update versions" commit to not bump the sdk and remove the last commit? would be cleaner to merge, as i don't want to squash all of them

**Comment by Domi04151309** [comment]
Sure if you can tell me how to do that because I don't.

**Comment by 92lleo** [comment]
Sure, no problem. Basic procedure is https://stackoverflow.com/a/1186549
The commit id you want to change is 0e7f88b
In the editor, where you have to change "pick" to "edit", you can remove the line "pick 3b5bdd0 Revert SDK bump" to remove the commit.
Then just follow procedure of the stack overflow answer and force push to this PR
(I can also do it if you want, but I can also support you in the process, just ping me if you have questions)

**Comment by Domi04151309** [comment]
Okay, I think that worked. Vim is really confusing if you've never used it before.

**Comment by 92lleo** [comment]
Yes, perfect, thanks! 
If you don't want to get into vim and only use a command line editor from time to time, I suggest to go with nano instead of vim

**Comment by Domi04151309** [comment]
I did use vim because the git shell automatically opened vim. Usually I'd use Nano on Linux but I did it on Windows.

**Comment by 92lleo** [comment]
In case you want to change it: If you run one of the current git-for-windows installs again, you can choose between vim, nano, (and anoter one?). Idk if it installs nano automatically, then you just could `git config --global core.editor "nano"`

**Comment by Domi04151309** [comment]
Thanks, I'll take a look at it

**Pull request merged**